
recent autoconf does not have AX_CODE_COVERATE and, although wget tries to
compensate, it doesn't allow enough.

diff -wpruN '--exclude=*.orig' a~/configure.ac a/configure.ac
--- a~/configure.ac	1970-01-01 00:00:00
+++ a/configure.ac	1970-01-01 00:00:00
@@ -34,6 +34,7 @@ dnl Error if AX_CODE_COVERAGE if missing
 m4_pattern_forbid([^_?AX_])
 m4_pattern_allow([AX_CHECK_GNU_MAKE_HEADLINE])
 m4_pattern_allow([_AX_CODE_COVERAGE_GCOV_PROG_WITH])
+m4_pattern_allow([AX_CODE_COVERAGE])
 
 AC_INIT([wget],[m4_esyscmd(build-aux/git-version-gen .tarball-version)],[bug-wget@gnu.org])
 AC_PREREQ([2.64])
